package.json

```diff
@@ -23,7 +23,7 @@
     "postshrinkwrap": "if test -z $CI; then \n npm run force-package-lock-https \n fi; \n git update-index --assume-unchanged .env",
```

Comment:
There are several other references to `http://localhost:7080` `http://localhost:9001` in this file & in the Cypress configuration `cypress/support/config.js` `local.env` file and in various unit test files for mocking.